### ISSUE TITLE
Caption background

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#1068]
### Caption background
What would be interesting is also to add the possibility of a background for captions.
Whether it's black or with opacity adjustable by the user:
- first, a black background (or another color at the user's convenience or a panel of proposed colors)
- second, opacity adjustable by the user.
- 
This would help facilitate reading for certain visual impairments in addition to hearing impairment.

**Comment by lebaudantoine** [comment]
Addressed in https://github.com/suitenumerique/meet/pull/1122 by @Ovgodd
